### PR TITLE
Skottie test

### PR DESCRIPTION
**Pull request by bodymovin**
Adds support for creating screenshots based on skottie

### CODE REVIEW

File: images/js/screenshot_skottie.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  const totalFrames = animationData.op - animationData.ip;
+  while (currentFrame < totalFrames) {
+    player.goToAndStop(currentFrame);
+    canvasSnapshot(
+      canvas,
+      snapshotsContainer,
+      animationData.w * renderSettings.resolution,
+      animationData.h * renderSettings.resolution,
+    );
+    await wait(1); // eslint-disable-line no-await-in-loop
```
Comment (fmalita) [inline images/js/screenshot_skottie.js:67]: I'm curious why this is needed?

Comment (bodymovin) [reply images/js/screenshot_skottie.js:67]: @fmalita it's a simple way of not blocking the main thread if all execution is synchronous. But It's probably fine if it's not there.

File: images/index.js [hunk 2]
```diff
@@ -64,27 +68,107 @@ const getSettings = async () => {
...
+  const encodingMap = {
+    js: 'utf8',
+    json: 'utf8',
+    html: 'utf8',
+  };
+  return (fileType) => encodingMap[fileType];
+})();
+
+const getContentTypeHeader = (() => {
+  const encodingMap = {
```
[the remark this answers is not in the document]
Comment (bodymovin) [reply images/index.js]: @fmalita fixed!

File: images/js/screenshot_skottie.js (hunk 1 above)
Comment (fmalita) [reply images/js/screenshot_skottie.js:67]: Makes sense, thanks.

**Review by fmalita — APPROVED** [review]

**Pull request merged**